html2docx/html2docx.py

```diff
@@ -183,8 +204,17 @@ def handle_starttag(self, tag: str, attrs: List[Tuple[str, Optional[str]]]) -> N
             self.init_run([("underline", True)])
         elif tag == "ul":
             self.add_list_style("List Bullet")
+        elif tag == "table":
+            self.init_table(attrs)
+        elif tag == "tr":
+            self.table_data.append([])
 
     def handle_data(self, data: str) -> None:
+        if self.tag in ("td", "th"):
```

Comment:
This is a list of values of homogeneous types, use a list. Tuples are usually meant for heterogeneous data. https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences:
> Though tuples may seem similar to lists, they are often used in different situations and for different purposes. Tuples are immutable, and usually contain a heterogeneous sequence of elements that are accessed via unpacking (see later in this section) or indexing (or even by attribute in the case of namedtuples). Lists are mutable, and their elements are usually homogeneous and are accessed by iterating over the list.

```suggestion
        if self.tag in ["td", "th"]:
```